### ISSUE TITLE
why mAP always zero? And a test problem

### ISSUE THREAD

**Issue by HHHcwb** [AlexeyAB/darknet#6754]
Hello, I used you model yolov4.conv.137 to detect smoke but After I train 3600 iterations, I got this:
2436
 detections_count = 0, unique_truth_count = 2604
class_id = 0, name = smoke, ap = 0.00%           (TP = 0, FP = 0)

 for conf_thresh = 0.25, precision = -nan, recall = 0.00, F1-score = -nan
 for conf_thresh = 0.25, TP = 0, FP = 0, FN = 2604, average IoU = 0.00 %

 IoU threshold = 50 %, used Area-Under-Curve for each unique Recall
 mean average precision (mAP@0.50) = 0.000000, or 0.00 %
Total Detection Time: 415 Seconds

Set -points flag:
 `-points 101` for MS COCO
 `-points 11` for PascalVOC 2007 (uncomment `difficult` in voc.data)
 `-points 0` (AUC) for ImageNet, PascalVOC 2010-2012, your custom dataset

 mean_average_precision (mAP@0.5) = 0.000000
New best mAP!

The dataset is manually annotated by myself, like this
![aug_99854019_1_358102511](https://user-images.githubusercontent.com/53850505/94440753-90dc8e00-01d4-11eb-9fd6-5a2a5c526c88.jpg)
 
![aug_55257966_2_1402743140](https://user-images.githubusercontent.com/53850505/94440730-8a4e1680-01d4-11eb-9e70-26311023c6a2.jpg)

And loss like this:
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.000000, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.000000, iou_loss = 0.000000, total_loss = 0.000000 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.000004, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.000014, iou_loss = 0.000000, total_loss = 0.000014 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.871772, GIOU: 0.871772), Class: 0.999187, Obj: 0.596893, No Obj: 0.003404, .5R: 1.000000, .75R: 1.000000, count: 2, class_loss = 0.329456, iou_loss = 0.318011, total_loss = 0.647467 
 total_bbox = 40, rewritten_bbox = 0.000000 % 

the region 161 looks like normal, because most of my data set is big target.
but I am confuse why my mAP is always zero, and when I test a picture it shows no detection even it has object.
Can anyone tell me what is going on or someone has the same issue like me? Thanks a lot

**Comment by Srikeshram** [comment]
I am also having this issue. Do you resolve @HHHcwb @AlexeyAB ?
